test/helpers/story/drivers/dapp-utils/aave.utils.ts

```diff
@@ -57,6 +62,34 @@ async function withdrawAave(
   aDaiBalance.eq(0).should.eql(true, '')
 }
 
+async function claimAave(
+  hre: HardhatRuntimeEnvironment,
+  loan: LoanHelpersReturn
+): Promise<void> {
+  const { contracts, network } = hre
+  const { details, diamond } = loan
+  const dappAddresses = getDappAddresses(network)
+  const IncentiveController = await contracts.get<IAaveIncentivesController>(
+    'IAaveIncentivesController',
+    {
+      at: dappAddresses.aaveIncentivesControllerAddress,
+    }
+  )
+  const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
+  const aaveBefore = await IncentiveController.getUserUnclaimedRewards(
+    escrowAddress
+  )
+  const assets = [details.lendingToken.address]
+  expect(BigNumber.from(aaveBefore).gt(0)).to.equal(true)
+  await diamond
+    .connect(details.borrower.signer)
+    .aaveClaimAave(details.loan.id, aaveBefore, assets)
+  const aaveAfter = await IncentiveController.getUserUnclaimedRewards(
+    escrowAddress
+  )
+  expect(aaveAfter.toString()).to.equal('0')
```

Comment:
be sure to check that the claimed balance went to the user's address

Reply:
done

Reply:
done